### DISCUSSION TITLE
Release Policy for Commercial Releases?

### DISCUSSION THREAD

**Discussion by davidvedvick** [emgucv/emgucv#777] [Ideas]
Hello,

I recently had an issue crop up from a missing dependency for `emgu.cv.runtime.windows.dldt`, it appears v4.5.4.4788 went missing. I assume this is because it's cheaper in some way to only maintain two releases in the private feed, or perhaps some other reason. While this is fine to me, I can't find any mention of this release policy on the documentation page, which would be helpful for educating others on why a new installer of our software failed.

Thanks!

**Comment by emgucv** [comment]
Sorry about that, due to the large size of the nuget release packages, our nuget server only have sufficient space to keep two releases. With the new 4.6.0 release being pushed to the repo, the 4.5.4 release is being retired. 

We will update the documentation to mention that only 2 releases will be keep at this moment. We are also looking at alternative nuget servers that can let us host more data.

**Reply by davidvedvick** [reply]
Great, thank you!
